### PR TITLE
add landsat sns

### PR DESCRIPTION
**Pull request by carioca-au**

### CODE REVIEW

File: data_specs/Landsat_C2_SR_specs.ipynb [hunk 1]
```diff
@@ -427,9 +427,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "|AWS S3 details | |\n",
+    "|AWS S3 details | PDS |\n",
```
Comment (alexgleith) [inline data_specs/Landsat_C2_SR_specs.ipynb:430]: PDS means nothing to others, so let's leave it out.

File: data_specs/Landsat_C2_SR_specs.ipynb [hunk 2]
```diff
@@ -444,6 +444,26 @@
...
+    "|----------|-------------|\n",
+    "| SNS ARN | `arn:aws:sns:af-south-1:543785577597:deafrica-landsat-scene-topic` |\n",
+    "| Region |`af-south-1` |\n",
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The sns are in the AWS region `af-south-1` (Cape Town). Additional region specifications can be applied as follows:\n",
```
Comment (alexgleith) [inline data_specs/Landsat_C2_SR_specs.ipynb:461]:
```suggestion
    "The SNS endpoints are in the AWS region `af-south-1` (Cape Town). Additional region specifications can be applied as follows:\n",
```

**Review by alexgleith — CHANGES_REQUESTED** [review]

File: data_specs/Landsat_C2_SR_specs.ipynb (hunk 2 above)
Comment (carioca-au) [reply data_specs/Landsat_C2_SR_specs.ipynb]: done

File: data_specs/Landsat_C2_SR_specs.ipynb [hunk 3]
```diff
@@ -448,7 +448,48 @@
...
+    "| Region |`af-south-1` |\n",
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The SNS endpoints are in the AWS region `af-south-1` (Cape Town). Additional region specifications can be applied as follows:\n",
+    "\n",
+    "`aws sns list-topics --region=af-south-1 `\n",
```
Comment (alexgleith) [inline data_specs/Landsat_C2_SR_specs.ipynb:484]: This won't work for anyone unless they're on our account.

Comment (carioca-au) [reply data_specs/Landsat_C2_SR_specs.ipynb:484]: removed

**Pull request closed without merging**